azkaban-common/src/main/java/azkaban/metrics/MetricsManager.java

```diff
@@ -28,24 +28,26 @@
 import java.lang.reflect.Constructor;
 import java.time.Duration;
 import java.util.concurrent.TimeUnit;
+import javax.inject.Inject;
```

Comment:
In general, try to make small incremental changes. When a change becomes bigger, it is more likely to miss issues. It takes longer to finish reviews too.